backend/apps/users/models.py

```
@@ -0,0 +1,117 @@
+    """ Custom user manager """ 
+    def create_user(self, email, name, password=None, user_type=None):
+        """ Create and return a user """
+        if not email:
+            raise ValueError("Users must have an email address")
+        email = self.normalize_email(email)
+        user = self.model(email=email, name=name, user_type=user_type)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, name, password=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_superuser`

(ANN201)